src/pool/srv_internal.h

```diff
@@ -56,14 +56,23 @@ struct pool_iv_map {
 	struct pool_buf	piv_pool_buf;
 };
 
+/* The structure to serialize the prop for IV */
 struct pool_iv_prop {
 	char		pip_label[DAOS_PROP_LABEL_MAX_LEN];
 	char		pip_owner[DAOS_ACL_MAX_PRINCIPAL_BUF_LEN];
 	char		pip_owner_grp[DAOS_ACL_MAX_PRINCIPAL_BUF_LEN];
 	uint64_t	pip_space_rb;
 	uint64_t	pip_self_heal;
 	uint64_t	pip_reclaim;
-	struct daos_acl	pip_acl;
+	struct daos_acl	*pip_acl;
+	d_rank_list_t   pip_svc_list;
+	uint32_t	pip_acl_offset;
+	uint32_t	pip_svc_list_offset;
+	char		pip_iv_buf[0];
```

Comment:
the pool IV operation is not frequent right? maybe need not to save the allocation times. just use "struct daos_acl	pip_acl; d_rank_list_t   pip_svc_list" and allocate internal buffer when needed maybe simpler and clearer.